### PR TITLE
Just a test bro

### PR DESCRIPTION
**Pull request by italofd**

### CODE REVIEW

**Comment by italofd** [comment]
Comment

File: src/__tests__/ui.test.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import Pivetada from "../components/index";
 import { screen } from "@testing-library/dom";
-import { render, within } from "@testing-library/react";
```
Comment (italofd) [inline src/__tests__/ui.test.tsx:4]: Not right

File: src/__tests__/ui.test.tsx [hunk 2]
```diff
@@ -28,8 +28,6 @@ describe("UI", () => {
 		 */
 		const text = screen.getByText(/Hello, Im Italo, a web developer/);
```
Comment (cptvictor95) [inline src/__tests__/ui.test.tsx:30]: italin talin lin

**Review by cptvictor95 — CHANGES_REQUESTED** [review]

File: src/__tests__/ui.test.tsx (hunk 2 above)
Comment (italofd) [reply src/__tests__/ui.test.tsx:30]: sure

**Comment by italofd** [comment]
Thick

File: src/__tests__/ui.test.tsx [hunk 3]
```diff
@@ -40,5 +38,8 @@ describe("UI", () => {
 		const link = within(div).getByRole("link", { name: "xoblas" });
```
Comment (cptvictor95) [inline src/__tests__/ui.test.tsx:39]: lalalala

**Review by cptvictor95 — CHANGES_REQUESTED** [review]

**Pull request merged**